qt/aqt/editor.py

```diff
@@ -696,7 +704,7 @@ def cleanup(self) -> None:
         # prevent any remaining evalWithCallback() events from firing after C++ object deleted
         if self.web:
             self.web.cleanup()
-            self.web = None
+            self.web = None  # type: ignore
```

Comment:
I did `# type: ignore` here instead of changing self.web's type to `EditorWebView | None` so that I didn't have to put a bunch of `assert self.web is not None`. However, I can do the latter if that's more desirable.

Reply:
In cases like this where the property ends up getting set as part of __init__() and unset only on cleanup, it's not very risky, and it avoids the extra assertion boilerplate. :+1: